src/Lykke.Service.ExchangeConnector/Exchanges/Concrete/BitMEX/BitMexExchange.cs

```diff
@@ -157,12 +168,138 @@ private static IReadOnlyList<Order> EnsureCorrectResponse(string id, object resp
         protected override void StartImpl()
         {
             OnConnected();
-            _orderBooksHarvester.Start();
+            _socketSubscriber.Start();
+            _measureTimer = new Timer(OnMeasureTimer, null, TimeSpan.Zero, TimeSpan.FromSeconds(60));
         }
 
         protected override void StopImpl()
         {
-            _orderBooksHarvester.Stop();
+            _socketSubscriber.Stop();
```

Comment:
I think methods below should be extracted to a dedicated class. They use nothing from the exchange class